mkposters/mkposter.py

```diff
@@ -68,10 +121,14 @@ def mkposter(datadir):
         if not (_here / "third_party" / "dart-sass" / "SASSBUILT.txt").exists():
             post_install(package_dir=str(_here))
 
+        style_sheet = f"{_here}/style.scss"
+        if scss:
+            style_sheet = scss
```

Comment:
I think it'd be better to look inside `datadir` for `style.scss`, just like how we currently look for `poster.md`.

Do you think it'd be better to have custom styling completely override the default styling (as is the case at the moment) or just override it?

Reply:
Worth noting that the current implementation will just override/ignore any existing user-specified file.

I think the best overall behaviour here would be to check if the user has a `style.scss` file inside `datadir`, and to have an additional `override_scss` flag passed to `mkposter`, defaulting to `False`. If `override_scss==False` then the user-specified `style.scss` is appended to the default `style.scss` prior to running dart-sass. If `override_scss=True` then the user-specified `style.scss` is used instead of the default `style.scss` when running dart-sass.